reward/src/lib.rs

```diff
@@ -0,0 +1,534 @@
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+
+use ethers::prelude::{Address, SignerMiddleware, U256};
+
+use librad::git::identities;
+use librad::git::identities::SomeIdentity::Person;
+use librad::PeerId;
+
+use std::path::Path;
+use std::str::FromStr;
+
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use rad_ens::resolver::PublicResolver;
```

Comment:
Maybe best to move this to `common::ethereum` so that we don't depend on another cli-command directly :man_shrugging: